tests/unit/module_utils/core/test_scrub_none_parameters.py

```diff
@@ -0,0 +1,47 @@
+from plugins.module_utils.core import scrub_none_parameters
```

Comment:
```suggestion
from __future__ import (absolute_import, division, print_function)
__metaclass__ = type

from ansible_collections.amazon.aws.plugins.module_utils.core import scrub_none_parameters
```

Reply:
Just for my understanding, why do we need this change?

Reply:
'plugins.module_utils' vs 'ansible_collections.amazon.aws.plugins.module_utils' this is mostly an artifact of the collections migration: collections now get their own namespace.

the future / metaclass boiler plate is the same as ever it's just ensuring python 2 and 3 behave a little bit more consistently.